.github/workflows/nightly.yml

```diff
@@ -30,13 +30,33 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
+  set-meta-releases:
+    # read the meta-release versions from the branches.json file
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    outputs:
+      meta-releases: ${{ steps.set-meta-releases.outputs.meta-releases }}
+    steps:
+      - name: install libs
+        id: set-meta-releases
+        uses: cylc/release-actions/set-meta-releases@v1
+
   deploy:
+    needs: set-meta-releases
     runs-on: ubuntu-latest
     timeout-minutes: 10
     defaults:
       run:
         # use login shell for conda activation
         shell: bash -leo pipefail {0}
+    strategy:
+      # run one job per meta-release
+      matrix:
+        meta_release: ${{ fromJson(needs.set-meta-releases.outputs.meta-releases) }}
+      # don't stop if one job fails
+      fail-fast: false
+      # only run one job at a time
+      max-parallel: 1
```

Comment:
Note we have multiple nightly builds now, one per meta-release, this line will keep the jobs running in sequence preventing conflict errors on the versions.json file. This is why `GITHUB_RUN_ID` is now in the commit message so that we don't delete the results of a previous job.